core-web/libs/sdk/react/src/lib/next/components/DotCMSEditableText/DotCMSEditableText.tsx

```diff
@@ -44,7 +44,9 @@ export function DotCMSEditableText({
     const editorRef = useRef<Editor['editor'] | null>(null);
     const [scriptSrc, setScriptSrc] = useState('');
     const [initEditor, setInitEditor] = useState(false);
-    const [content, setContent] = useState(contentlet?.[fieldName] || '');
+    const [content, setContent] = useState(
+        contentlet?.[fieldName as keyof DotCMSBasicContentlet] || ''
```

Comment:
I think you can type the prop, to avoid doing the `as` 
try this in angular too